src/api/auth_backends.py

```diff
@@ -158,5 +158,5 @@ def authenticate(self, request, *args, **kwargs):
             pass
 
         if jwt_token:
-            return super().authenticate(request, *args, **kwargs)
+            return super().authenticate(request)
```

Comment:
Why remove `*args` and `**kwargs`?

Reply:
The [`authenticate`](https://github.com/data-mermaid/mermaid-api/blob/master/src/api/auth_backends.py#L31) method signature from `JWTAuthentication` parent class does not define those, unless I am missing something.

Reply:
Good point.